src/unity/toolkits/activity_classification/activity_classifier.cpp

```diff
@@ -145,19 +146,49 @@ void activity_classifier::init_options(
   add_or_update_state(flexmap_to_varmap(options.current_option_values()));
 }
 
+std::tuple<float, float> activity_classifier::compute_validation_metrics(
```

Comment:
Let's at least add a quick comment documenting what the semantics of the two tuple values are (accuracy and loss)